tests/nnstreamer_filter_extensions_common/unittest_tizen_template.cpp

```diff
@@ -202,14 +208,18 @@ TEST (nnstreamer_EXT_NICK_NAME_basic_functions, get_dimension)
   ret = sp->open (&prop, &data);
   EXPECT_EQ (ret, 0);
 
-  /** get input/output dimension successfully */
-  ret = sp->getInputDimension (&prop, &data, &res);
-  EXPECT_EQ (ret, 0);
+  /** python subplugin modifies methods based on model file */
+  if (sp->getInputDimension && sp->getOutputDimension) {
+    /** get input/output dimension successfully */
+    ret = sp->getInputDimension (&prop, &data, &res);
```

Comment:
One question: in most sub-plugins, `getInputDimension` and `getOutputDimension` copy tensors info using `gst_tensors_info_copy`. It means `g_strdup` could be performed if `name` exists. So, is it okay just to override `res`?

Reply:
Added `gst_tensors_info_free` to overcome the mem-leak of name.